scalardb-analytics-postgresql-sample/README.md

```diff
@@ -1,19 +1,40 @@
-# Sample Application for ScalarDB Analytics with PostgreSQL
+# Create a Sample Application That Uses ScalarDB Analytics with PostgreSQL
 
-This tutorial describes how to create a sample application for ScalarDB Analytics with PostgreSQL.
+This tutorial describes how to create a sample application that uses ScalarDB Analytics with PostgreSQL.
+
+## Overview
+
+The sample application shows how you can run two types of queries: a single-table query and a multi-table query.
+
+### What you can do in this sample application
+
+The sample application supports the following types of transactions:
+
+- Read data and calculate summaries.
+- Join tables that span multiple storages.
+
+{% capture notice--info %}
+**Note**
+
+You can run any arbitrary query that PostgreSQL supports on the imported tables in the sample application. Since ScalarDB Analytics with PostgreSQL supports all queries that PostgreSQL supports, you can use not only join, aggregation, filtering, and ordering as shown in the example, but also the window function, lateral join, or various analytical operations.
+
+To see which types of queries PostgreSQL supports, see the [PostgreSQL documentation](https://www.postgresql.org/docs/current/index.html).
+{% endcapture %}
+
+<div class="notice--info">{{ notice--info | markdownify }}</div>
 
 ## Prerequisites
 
-- Docker
-- psql
+- [Docker](https://www.docker.com/get-started/) 20.10 or later with [Docker Compose](https://docs.docker.com/compose/install/) V2 or later
+- [psql](https://www.postgresql.org/docs/current/app-psql.html)
 
-## Set up the database
+## Set up ScalarDB Analytics with PostgreSQL
 
-First, you must set up the database to run analytical queries with ScalarDB Analytics with PostgreSQL. If you have not set up the database yet, please follow the instructions in [Getting Started](https://scalardb.scalar-labs.com/docs/latest/scalardb-analytics-postgresql/getting-started).
+First, you must set up the database to run analytical queries with ScalarDB Analytics with PostgreSQL. If you haven't set up the database yet, please follow the instructions in [Getting Started](https://scalardb.scalar-labs.com/docs/latest/scalardb-analytics-postgresql/getting-started).
 
-## Schema in ScalarDB
+### Schema in ScalarDB
 
-In this application, you have tables with the following schema in the ScalarDB database:
+In the application, you have tables with the following schema in the ScalarDB database:
```

Comment:
In this sentence, "ScalarDB database" means the database that ScalarDB virtually provides, not the underlying physical databases that ScalarDB manages, like MySQL or Cassandra. I'm unsure how to call them by different names precisely, but I tried to be consistent with the names in this document. Do you have any idea on how to call them?

Reply:
I read https://www.vldb.org/pvldb/vol16/p3768-yamada.pdf again. The paper doesn't mention how to call database that ScalarDB virtually provides, but it often calls the physical database managed by ScalarDB "underlying database"
<img width="350" alt="image" src="https://github.com/scalar-labs/scalardb-samples/assets/59043/f804af06-3c57-4def-ba4a-efe6f9645a80">.

So, how about this?
- database that ScalarDB virtually provides => simply "ScalarDB database" (actually we're used to this term somehow?)
- underlying physical database managed by ScalarDB => "underlying database"

Reply:
Yeah, I agree with the @komamitsu 's idea. While it would be easier for us to understand what "virtual" means, AFAIR, I haven't seen the word "virtual database" to mention the database that ScalarDB provides.

Reply:
Understood, thank you! I've noted this for future reference and will leave this wording as is🙏